tests/unit/test_pep425tags.py

```diff
@@ -159,3 +159,22 @@ def test_manylinux1_tag_is_first(self):
                 assert arches == ['manylinux1_x86_64', 'linux_x86_64', 'any']
             else:
                 assert arches == ['manylinux1_x86_64', 'linux_x86_64']
+
+    def test_manylinux1_check_glibc_version(self):
```

Comment:
Ah, right, I see.

Does pip have any facilities for testing warnings, or do I have to roll my own?